### PR TITLE
Document Windows Service instrumentation

### LINKED ISSUE #661: Document how to instrument a Windows Service

### PR DESCRIPTION
**Pull request by pellared**
Fixes #661

Changes proposed in this pull request:
- Document how to instrument a Windows Service

### CODE REVIEW

File: docs/windows-service-instrumentation.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+```env
+Var1=Value1
+Var2=Value2
+```
+
+Below is an example how instrumentation can be configured in PowerShell:
+
+```powershell
+$installationLocation = "C:\some\dir" # The path where you put the OTel .NET AutoInstrumentation binaries
```
Comment (Kielek) [inline docs/windows-service-instrumentation.md:29]: Do we have any recommended path for installation folder?

File: docs/windows-service-instrumentation.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+The environment variables can be defined
+in a `REG_MULTI_SZ` (multiline registry value) called `Environment`
+in the following format:
+
+```env
+Var1=Value1
+Var2=Value2
+```
+
+Below is an example how instrumentation can be configured in PowerShell:
```
Comment (Kielek) [inline docs/windows-service-instrumentation.md:26]: Can it be execeuted without elevated privileges (Administrator account)?

**Review by Kielek — APPROVED** [review]

File: docs/windows-service-instrumentation.md (hunk 1 above)
Comment (pellared) [reply docs/windows-service-instrumentation.md:29]: No.

File: docs/windows-service-instrumentation.md (hunk 2 above)
Comment (pellared) [reply docs/windows-service-instrumentation.md:26]: No. Setting value in HKLM(HKEY_LOCAL_MACHINE) requires admin permissions. Any proposal on how to word it?

```suggestion
Below is an example how instrumentation can be configured in PowerShell (run as administrator):
```
?

**Review by rajkumar-rangaraj — APPROVED** [review]

**Review by pjanotti — APPROVED** [review]
LGTM

**Pull request merged**